### PR TITLE
GEOSEARCH and GEOSEARCHSTORE

### PR DESCRIPTION
**Pull request by AvitalFineRedis**
### Pull Request check-list

_Please make sure to review and check all of these items:_

- [x] Does `$ tox` pass with this change (including linting)?
- [x] Do the CI tests pass with this change (enable it first in your forked repo and wait for the github action build to finish)?
- [x] Is the new or changed code fully tested?
- [x] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?

_NOTE: these things are not required to open a PR and can be done
afterwards / while the PR is open._

### Description of change

_Please provide a description of the change here._

### CODE REVIEW

**Comment by AvitalFineRedis** [comment]
@chayim

File: tests/test_commands.py [hunk 1]
```diff
@@ -1115,7 +1115,7 @@ def test_linsert(self, r):
         assert r.lrange('a', 0, -1) == [b'1', b'2', b'2.5', b'3']
         assert r.linsert('a', 'before', '2', '1.5') == 5
         assert r.lrange('a', 0, -1) == \
-            [b'1', b'1.5', b'2', b'2.5', b'3']
+               [b'1', b'1.5', b'2', b'2.5', b'3']
```
Comment (chayim) [inline tests/test_commands.py:1118]: Is the green highlight a tab? If so - and for the others can we replace tabs with spaces. If not.. I'll see why it's highlighting this way.

Comment (AvitalFineRedis) [reply tests/test_commands.py:1118]: It looks like spaces

**Comment by AvitalFineRedis** [comment]
#1546

**Pull request merged**